template/package.json

```diff
@@ -14,8 +14,8 @@
     "boilerplate"
   ],
   "dependencies": {
+    "@sailshq/lodash": "^3.10.3",
```

Comment:
Why is lodash this scoped to @sailshq? Do they use a fork?

Reply:
seems so, I generated a new Sailsjs project with the latest version and noticed they're using the scoped version. Not sure what the reason.

Reply:
yes. The project failed to run without it.

Reply:
Ah yes I see it's being required in the config file. That's fine.